xml/Microsoft.Extensions.FileProviders.Composite/CompositeDirectoryContents.xml

```diff
@@ -153,9 +153,17 @@
       </ReturnValue>
       <Parameters />
       <Docs>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <summary>Returns an enumerator that iterates through a collection.</summary>
+        <returns>An <see cref="T:System.Collections.IEnumerator" /> object that can be used to iterate through the collection.</returns>
```

Comment:
Let's stay consistent with the rest of the files and use natural language to refer to the enumerator object (plus, natural language is preferred):
```suggestion
        <returns>An enumerator object that can be used to iterate through the collection.</returns>
```